gemrb/GUIScripts/pst/GUIOPT.py

```diff
@@ -416,22 +413,35 @@ def OpenQuitMsgWindow ():
 	# Cancel
 	Button = Window.GetControl (2)
 	Button.SetText (4196)
-	Button.SetEvent (IE_GUI_BUTTON_ON_PRESS, OpenQuitMsgWindow)
+	Button.SetEvent (IE_GUI_BUTTON_ON_PRESS, CloseQuitMsgWindow)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['IE_GUI_BUTTON_ON_PRESS' may be undefined, or defined from star imports: GUIDefines](https://app.codacy.com/app/gemrb/gemrb/pullRequest?prid=4591638)